### PR TITLE
Gui 2764 - remove eucaroutes

### PR DESCRIPTION
**Pull request by dkavanagh**
https://eucalyptus.atlassian.net/browse/GUI-2764

### CODE REVIEW

File: eucaconsole/static/js/services/alarm-service.js [hunk 1]
```diff
@@ -5,114 +5,93 @@
...
-                        url: path
-                    }).then( function (response) {
-                        return response.data || {
-                            alarm: {}
-                        };
-                    });
-                });
+            return $http({
+                method: 'GET',
+                url: '/alarms/' + btoa(id) + '/json'
```
Comment (coyote240) [inline eucaconsole/static/js/services/alarm-service.js]: This is space for an injection attack.  $interpolate may be a bit safer.

File: eucaconsole/static/js/services/alarm-service.js [hunk 2]
```diff
@@ -5,114 +5,93 @@
...
-                    }).then(function (response) {
-                        var data = response.data || {
-                            history: []
-                        };
-                        return data.history;
-                    });
-                });
+            return $http({
+                method: 'GET',
+                url: '/alarms/' + btoa(id) + '/history/json'
```
Comment (coyote240) [inline eucaconsole/static/js/services/alarm-service.js]: Same as above, re: $interpolate

File: eucaconsole/static/js/services/alarm-service.js [hunk 3]
```diff
@@ -5,114 +5,93 @@
...
-                            'resource-type': type
-                        }
-                    }).then(function success (response) {
-                        var data = response.data.results || [];
-                        return data;
-                    });
-                });
+            return $http({
+                method: 'GET',
+                url: '/alarms/resource/' + id + '/json',
```
Comment (coyote240) [inline eucaconsole/static/js/services/alarm-service.js]: $interpolate.

File: eucaconsole/static/js/services/bucket-cors-service.js [hunk 4]
```diff
@@ -5,36 +5,30 @@
...
-                        url: path,
-                        data: {
-                            csrf_token: csrfToken,
-                            cors_configuration_xml: corsConfigXml
-                        }
-                    });
-                });
+            return $http({
+                method: 'PUT',
+                url: '/buckets/' + bucketName + '/cors_configuration',
```
Comment (coyote240) [inline eucaconsole/static/js/services/bucket-cors-service.js]: $interpolate

File: eucaconsole/static/js/services/alarm-service.js (hunk 1 above)
Comment (dkavanagh) [inline eucaconsole/static/js/services/alarm-service.js]: I see now that it prefixes {{ with backslash, so yes. Agreed.

**Pull request merged**